app/styles/templates/orders/_messages.scss

```diff
@@ -0,0 +1,78 @@
+body{
```

Comment:
couple of things : 

It's important to scope all the css rules to the page you're building, so make sure to put all those rules under a class/ID of your page. It limits the risks.

e.g
```
.order-chat-page {
 // all rules here
}
```

right now it's in the global scope, meaning if on some completely different page I use a similar class (e.g `message-section` is fairly generic), it would apply styles that I don't want.

Another thing is the `body { }` rule. That's quite dangerous cause it'll apply it for the whole app (potentially unwanted), better to have that rule on the container div of you page

Reply:
Yeah you're right. I'll do that. Thanks @patrixr